src/main/java/io/nats/client/impl/NatsDispatcher.java

```diff
@@ -164,40 +204,68 @@ public Dispatcher subscribe(String subject, String queueName) {
         if (queueName == null || queueName.length() == 0) {
```

Comment:
Can you add the subject == null check to this one, the others have it. Or we could move that check to the impl and just check queues and handlers as necessary

Reply:
It's there, I think the github collapsed diff hid it from view, but it's there.